packages/grid/x-data-grid-pro/src/internals/components/GridColumnPinningMenuItems.tsx

```diff
@@ -1,10 +1,9 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import MenuItem from '@mui/material/MenuItem';
-import { useGridApiContext } from '../../../hooks/utils/useGridApiContext';
-import { GridColDef } from '../../../models/colDef/gridColDef';
-import { GridPinnedPosition } from '../../../models/api/gridColumnPinningApi';
-import { GridApiPro } from '../../../models/api/gridApiPro';
+import { GridColDef } from '@mui/x-data-grid';
```

Comment:
I thought we would have a Pro `GridColDef`, instead of using module augmentation.

As example:

```suggestion
import { GridColDef } from '../../../gridColDef';
```

But we can see how this approach works in the long term.

Reply:
I think we should have a pro `GridColDef` at once point
But it was creating too many problems with the `api` params deep inside some `GridColDef` callbacks.
So I went for the easy way for now.